src/lib/create-config.ts

```diff
@@ -106,6 +106,11 @@ const defaultDbOptions: IDBOption = {
 
 const defaultSessionOption: ISessionOption = {
     ttlHours: safeNumber(process.env.SESSION_TTL_HOURS, 48),
+    clearSiteDataOnLogout: safeBoolean(
```

Comment:
This will probably need an update, `safeBoolean` was renamed to `parseEnvVarBoolean` in this PR: https://github.com/Unleash/unleash/pull/1642/files

Reply:
Nice!